pyproject.toml

```diff
@@ -33,6 +33,8 @@ fastjsonschema = { version = "^2.14.5", optional = true }
 pydantic = { version = "^1.8.2", optional = true }
 boto3 = { version = "^1.20.32", optional = true }
 typing-extensions = "^4.6.2"
+opentelemetry-sdk = "^1.18.0"
+opentelemetry-exporter-otlp-proto-grpc = "^1.18.0"
```

Comment:
Note: after the final tests with these providers (Datadog + OTEL) we must remove the providers and these dependencies.

Please don't resolve this conversation.

Reply:
Yeah that's true. I was thinking about the same thing